src/test/java/core/basesyntax/service/RegistrationServiceTest.java

```diff
@@ -0,0 +1,84 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exceptions.NotEnoughAgeForUsingStorageException;
+import core.basesyntax.exceptions.UserDoesNotExistException;
+import core.basesyntax.exceptions.WrongLoginException;
+import core.basesyntax.exceptions.WrongPasswordException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceTest {
+    private static final RegistrationService registrationService = new RegistrationServiceImpl();
+    private static final StorageDao storageDao = new StorageDaoImpl();
+
+    private static User validUser;
+    private static User userWithInvalidLogin;
+    private static User userWithInvalidPassword;
+    private static User userWithInvalidAge;
+    private static User userDoesNotExist;
+
+    @BeforeAll
+    static void beforeAll() {
+        validUser = createUser(13652827L, "qazwsxedc82738", 23, "qwdfrw34");
+        userDoesNotExist = createUser(1221L, "qazwhhff738", 23, "qwdfrw34");
+        userWithInvalidLogin = createUser(0L, "bob", 18, "qwdfrw");
+        userWithInvalidPassword = createUser(0L, "bobik34", 32, "wefwf");
+        userWithInvalidAge = createUser(0L, "borisik", 17, "qwdqwerty");
+    }
+
+    @Test
+    void shouldThrowUserDoesNotExistExceptionWhenUserDoesNotExistInStorage() {
+        Exception exception = assertThrows(UserDoesNotExistException.class, () ->
+                registrationService.register(userDoesNotExist));
+        assertEquals("User was not founded", exception.getMessage());
+    }
+
+    @Test
+    void shouldThrowWrongLoginExceptionWhenLoginLessSixCharacters() {
+        assertThrows(WrongLoginException.class,
+                () -> registrationService.register(userWithInvalidLogin));
+    }
```

Comment:
in one test method, you can test a couple users with invalid login, so please add such test users to cover edge cases (same with other test methods)
![image](https://github.com/mate-academy/jv-registration-validation/assets/48316721/cdec6152-e268-4677-a344-6e338045c94e)